### DISCUSSION TITLE
downloading scripts off github

### DISCUSSION THREAD

**Discussion by laurel-nave-powers** [sr320/course-fish546-2021#67] [Q&A] [answered]
I have my data all ready to go for input in a script. The scripts are on GitHub (https://github.com/calderatta/ca-exon-capture/tree/master/pipeline_scripts/data_preparation) but I don't understand how to download the scripts so that I can start to use them and modify them to my own project. Google doesn't seem to be getting me anywhere. Thank you for your help!

**Comment by kubu4** [comment, marked as answer]
The way with the least complicated explanation would be to clone the entire repo to your computer and then navigate to the files. You can do this by clicking on the green "Code" button on this page: https://github.com/calderatta/ca-exon-capture

The more verbose explanation which will just download the script files:

1. Click on the name of the file.

2. Right-click on the button labeled "Raw" (towards the right side of the page) and select "Download link as..." (the wording on this might differ, depending on which browser you're using).

3. Repeat for each file.

**Reply by laurel-nave-powers** [reply]
Thank you so much that worked! I think I tried everything but those options haha.
